src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx

```diff
@@ -63,6 +63,13 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
         onBlur={handleBlur}
       />
 
+      {/* render email error separately to have its design aligned with PhoneNumberInput */}
+      {errors.email && values.email && (
```

Comment:
I suppose we also want to show the error if the field was touched(after input loses focus)